lib/profile/edit_profile/sections/education/save_education.dart

```diff
@@ -38,16 +46,12 @@ class _SaveEducationState extends State<SaveEducation> {
     isStartYearFocused = false;
     isEndYearFocused = false;
 
-    // Temporary workaround until we use global state management
-    education = {
-      'school': widget.editMode ? widget.schoolInfo['school'] : null,
-      'degree': widget.editMode ? widget.schoolInfo['program'] : null,
-      'startYear': widget.editMode ? widget.schoolInfo['startYear'] : null,
-      'endYear': widget.editMode ? widget.schoolInfo['endYear'] : null
-    };
-
     textFieldFocusListener(schoolFocus, 'school');
     textFieldFocusListener(degreeFocus, 'degree');
+
+    if (!widget.editMode) {
+      generatedId = randomAlphaNumeric(10);
+    }
```

Comment:
i thought each `school`, `job` and `skill` item should have a unique ID so they're easier to access. This is how we're generating the ID for now but we can defs change this as we go along